src/dev/impl/DevToys/Strings/id-ID/JsonYaml.resw

```diff
@@ -144,17 +144,17 @@
   <data name="InvalidYaml" xml:space="preserve">
     <value>YAML yang diinputkan tidak sesuai</value>
   </data>
-  <data name="JsonToYaml" xml:space="preserve">
-    <value>Json ke Yaml</value>
+  <data name="Json" xml:space="preserve">
+    <value>Json</value>
```

Comment:
Maybe we should decide now if it should be presented as `Json` or `JSON` in all languages, because now there is no consistency. I vote for `JSON` and `YAML`, as `JSON` is `JavaScript Object Notation` and `YAML` is `Yet Another Markup Language`.